atomic_reactor/utils/cachito.py

```diff
@@ -147,14 +147,16 @@ def wait_for_request(
 
             # All other states are expected to be transient and are not checked.
 
-            elapsed = time.time() - start_time
+            updated_time = datetime.fromisoformat(response_json["updated"])
```

Comment:
While `fromisoformat` was introduced in python 3.7, atomic-reactor still supports python 2.7 (and it would also be nice to keep 3.6 compatibility since we plan on supporting CentOS 8 for buildroots soon).

Reply:
Thanks for catching that. I changed it to use `strptime` instead.